server.go

```diff
@@ -254,9 +255,6 @@ func (s *Server) Start() error {
 
 		log.Infof("RPC server listening on %s", s.grpcListener.Addr())
 		if s.restListener != nil {
-			s.restListener = tls.NewListener(
```

Comment:
hm, why was this here?

Reply:
I wondered about this myself... Should I remove it in a separate commit to make it more clear this was wrong in the first place?

Reply:
Yes 👍

Reply:
Done.